sdk/template/Azure.Template/samples/Sample1_CreateResource.md

```diff
@@ -0,0 +1,46 @@
+# Sample readme template
+
+Use the guidelines in each section of this template to write samples, for an example following this template.
+
+This sample shows how to detect all the anomalies in the entire time series.
+
+To get started, make sure you have satisfied all the prerequisites and got all the resources required by [README][README].
+
+**write the sample code snippets as following example**
+
+## Detect anomalies of the entire series
```

Comment:
Is it confusing that we're using examples from different service libraries in the template?  Would it make sense to stick to one service to illustrate that all the parts of the template work together for a single service SDK?